### ISSUE TITLE
Breach of Licensing

### ISSUE THREAD

**Issue by alectrocute** [ExistentialAudio/BlackHole#371]
Found this app which allegedly builds upon Blackhole’s source without proper licensing, thought I would post here so it is addressed.

https://www.gingeraudio.com/

Thanks.

**Comment by devinroth** [comment]
Thanks for bringing it to my attention.

Where does it say it’s building upon BlackHole? Obviously the driver looks exactly the same but I need some solid evidence before contacting them.

Regardless it’s clearly a copycat.

Devin Roth

**Comment by gchilds** [comment]
It looks to be written in c++, while BlachHole is c if I'm not mistaken.

How is it a copycat? People have been writing loopback audio drivers for the mac for as long as apple has been ignoring the need for them.

**Comment by devinroth** [comment]
First off, BlackHole is a copycat of Soundflower. And currently BlackHole is the only working open source option now that both Soundflower and Jack no longer work on the latest versions of macOS. That's what makes BlackHole unique compared to any other solutions.

Yes there are a number of loopback drivers out there but none are as blatantly a copycat of any as GroundControl is of BlackHole. Take a look at the website https://www.gingeraudio.com vs https://existential.audio/blackhole/. 

Naming: GroundControl 2ch vs BlackHole 2ch, GroundControl 16ch vs BlackHole 16ch. I took this idea from Soundflower but they use (2ch) instead. Even the name GroundControl is a riff off the space theme I'm giving everything Existential Audio. 

Email List: Asking people to subscribe to download the installer. Again not an original idea of mine, but no other loopback driver does this.

Color: Black background with white text. Same deal as above. 

Copy: "Route audio between applications" vs "Route audio between apps". That was 100% original by me. If you compare the copy on the websites you'll see that the similarities are more than just a coincidence. 

Without a doubt https://www.gingeraudio.com took a look at my website and design and just riffed off of it instead of coming up with their own ideas. Not that my ideas are all original since I borrowed them from a number of sources but it's not like I just took one source like Jack Router, or Loopback, or any of the other examples out there and just tried to clone them.

None of this is illegal unless they are using some of BlackHole's source. It's just lazy.
